src/components/Generator.vue

```diff
@@ -153,6 +155,18 @@ async function downloadZip() {
       </select>
     </p>
 
+    <details>
+      <summary>Advanced Options</summary>
+      <br/>
+      <div style="margin-left: 1rem">
```

Comment:
The style should ideally not be in the html. I would try to put it in a CSS file if possible. Unfortunately the component-local style probably doesn't work with the current website embedding setup.